src/librustdoc/html/render.rs

```diff
@@ -1908,6 +1911,7 @@ impl fmt::Display for AllTypes {
         print_entries(f, &self.derives, "Derive Macros", "derives")?;
         print_entries(f, &self.functions, "Functions", "functions")?;
         print_entries(f, &self.typedefs, "Typedefs", "typedefs")?;
+        print_entries(f, &self.trait_aliases, "Trait Aliases", "trait-alias")?;
```

Comment:
trait-aliases, to be consistent.

(should there be a hyphen at all?)

Reply:
It's a css class so I guess it's fine.

Reply:
It's nice to be consistent, also: y'all have theming support so the css classes do matter, if only a little bit